src/qtt/pgeometry.py

```diff
@@ -1755,37 +1755,38 @@ def ff(xx, figx=fig): return mpl2clipboard(fig=figx)
 # %%
 
 
-try:
-    def monitorSizes(verbose=0):
-        """ Return monitor sizes """
-        _qd = QtWidgets.QDesktopWidget()
-        if sys.platform == 'win32' and _qd is None:
-            import ctypes
-            user32 = ctypes.windll.user32
-            wa = [
-                [0, 0, user32.GetSystemMetrics(0), user32.GetSystemMetrics(1)]]
+def monitorSizes(verbose: int = 0) -> List[List[int]]:
+    """ Return monitor sizes
+
+    Args:
+        verbose: Verbosity level
+    Returns:
+        List with for each screen a list x, y, width, height
+    """
+    _qd = QtWidgets.QDesktopWidget()
```

Comment:
This line and the check if _qd is None in the next line is not needed
_qd is not used when win32 and in the else _qd is set again in all cases